README.md

```diff
@@ -141,36 +141,7 @@ For bigger changes, or if in doubt, make sure to talk about your contribution to
 via an issue, GitHub discussion, or reach out to the team either using the
 [Discord server](https://discord.gg/pxrBmy4).
 
-
-## Getting started
```

Comment:
Why are you removing this section?

Reply:
I wanted this section to be in the space shooter README.md.  Now thinking about it, this section would still be useful in the main README.md. I will add it again.